diff --git a/src/components/progressPigs/ProgressBar.js b/src/components/progressPigs/ProgressBar.js
index 32a9a55..8b2b5db 100644
--- a/src/components/progressPigs/ProgressBar.js
+++ b/src/components/progressPigs/ProgressBar.js
@@ -5,18 +5,13 @@ import pigImage from "../../assets/pig.png";
 const ProgressBar = (props) => {
   const {bgcolor, completed} = props;
   const containerStyles = {
-    height: 20,
     width: '80%',
     backgroundColor: "#e0e0de",
     borderRadius: 50,
-    marginLeft: 50,
-    marginRight: 50,
-    marginTop: '-7%',
-    marginBottom: '10px',
     position: "relative",
+      margin: "20em auto 0 auto",
   }
   const fillerStyles = {
-    height: '100%',
     width: `${completed}%`,
     backgroundColor: "#3A693F",
     borderRadius: 'inherit',
@@ -29,53 +24,38 @@ const ProgressBar = (props) => {
     fontWeight: 'bold'
   }
   const pigImgStyles = {
-    width: '100%'
+    width: '100%',
+      position: "absolute",
+      left: "1em",
+      top: "-10em",
   }
   const pigStyles = {
-    width: '40%',
-    position: 'relative',
-    textAlign: 'center',
-    color: 'white',
-    left: `${completed-30}%`
-  }
-  const compStyles = {
-    position: 'absolute',
-    textAlign: 'center',
-    marginLeft: 50,
-    top: '37%',
-    left: '33%'
-    
-  }
-  const savedStyles = {
+    width: '80%',
+      marginLeft: "-40%",
     position: 'absolute',
-    textAlign: 'center',
-    marginLeft: 50,
-    top: '47%',
-    left: '33%'
+    color: 'white',
+    left: `${completed}%`,
+      bottom: "10em",
+      textAlign: "center",
   }
-  const totalStyles = {
-    position: 'absolute',
-    textAlign: 'center',
-    marginLeft: 50,
-    top: '57%',
-    left: '33%'
+  const pigdetails = {
+      position: "relative",
+      zIndex: 10,
   }
 
   return (
-    <div>
+      <div style={containerStyles}>
       <div class="container pig-stats" style={pigStyles}>
               <img src={pigImage} style={pigImgStyles}/>
-              <p class="centered" style={compStyles}> {props.completed}% complete </p>
-              <p class="centered" style={savedStyles}> ${props.current} saved </p>
-              <p class="centered" style={totalStyles}> ${props.total} to go </p>
+              <p style={pigdetails} > {props.completed}% complete </p>
+              <p style={pigdetails} > ${props.current} saved </p>
+              <p style={pigdetails} > ${props.total} to go </p>
       </div>
-      <div style={containerStyles}>
       {props.children}
         <div style={fillerStyles}>
           <span style={labelStyles}>{`${completed}%`}</span>
         </div>
       </div>
-    </div>
   );
 };
 export default ProgressBar;
